featuretools/tests/primitive_tests/test_transform_features.py

```diff
@@ -1133,10 +1133,12 @@ def test_cfm_with_lag_and_non_nullable_column(pd_es):
 
     periods = 5
     lag_primitive = NumericLag(periods=periods)
+    cutoff_times = pd_es['new_log'][['id', 'datetime']]
```

Comment:
@rwedge Was this what you had in mind for adding individual cutoff times for each row?

Reply:
yep!